tests/suites/tenant/initialLoad.test.ts

```diff
@@ -0,0 +1,55 @@
+import {expect, test} from '@playwright/test';
+
+import {tenantName} from '../../utils/constants';
+
+import {TenantPage} from './TenantPage';
+
+const pageQueryParams = {
+    schema: tenantName,
+    name: tenantName,
+    tenantPage: 'diagnostics',
+};
+
+test.describe('Tenant initial load', () => {
+    test('Tenant diagnostics page is visible', async ({page}) => {
+        const tenantPage = new TenantPage(page);
+        await tenantPage.goto(pageQueryParams);
+
+        await expect(page.locator('.kv-tenant-diagnostics')).toBeVisible();
+    });
+
+    test('Tenant diagnostics page is visible when describe returns no data', async ({page}) => {
+        await page.route('http://localhost:8765/viewer/json/describe?*', async (route) => {
```

Comment:
Please, make host a variable, it should be possible easily run these tests on some other stand, not only on the stand that is run by `npm run dev` command